zhaquirks/tuya/ts0601_energy_meter.py

```diff
@@ -0,0 +1,1261 @@
+"""Tuya Energy Meter."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from typing import Any, Final
+
+from zigpy.quirks.v2.homeassistant import PERCENTAGE, EntityType, UnitOfTime
+import zigpy.types as t
+from zigpy.zcl import Cluster
+from zigpy.zcl.clusters.homeautomation import MeasurementType
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+from zhaquirks import LocalDataCluster
+from zhaquirks.tuya import (
+    DPToAttributeMapping,
+    PowerPhaseVariant2,
+    PowerPhaseVariant3,
+    TuyaLocalCluster,
+    TuyaZBElectricalMeasurement,
+    TuyaZBMeteringClusterWithUnit,
+)
+from zhaquirks.tuya.builder import TuyaQuirkBuilder
+from zhaquirks.tuya.mcu import TuyaMCUCluster
+
+ENERGY_DIRECTION: Final = "energy_direction"
+
+
+class Channel(t.enum8):
```

Comment:
Why is this an enum?
From my testing, it adds confusion on the unique_id as it ends up with a string-formatted enum rather than a simple integer.